ui/src/templates/components/TemplatesPage.tsx

```diff
@@ -35,7 +35,7 @@ interface StaticTemplate {
 
 const staticTemplates: StaticTemplate[] = _.map(statics, (template, name) => ({
   name,
-  template,
+  template: template as TemplateSummary,
```

Comment:
Not sure how this made it onto master. This PR shouldn't be effecting the template type, issue likely stems from an earlier PR on master that didn't set off the linter and should be addressed when Templates are refactored